oop/music/electro.py

```diff
@@ -0,0 +1,22 @@
+from oop.music.instrumental_song import Instrumental
+
+class Electro(Instrumental):
+    """A class that defines all music in the electro genre"""
+
+    # Assigning all properties of Music and Instrumental to the Electro class
+    # Instances of Electro have a synthetizer as a leading instrument
+    def __init__(self, title: str, bpm: int, instrument: [], leading_instrument: str, has_lyrics: bool):
+        super().__init__(title, bpm, instrument, leading_instrument, has_lyrics)
+
+        self.leading_instrument = leading_instrument
+        leading_instrument = 'synthetizer'
+        self.has_lyrics = has_lyrics
+        has_lyrics = False
```

Comment:
similar situation to the above - possible confusion here around data membership. has_lyrics user variable is being reset to False.